packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -426,6 +426,13 @@ const double _kDefaultFontSize = 14.0;
 ///
 /// ### Inconsistent platform fonts
 ///
+/// By default, fonts differ depending on the platform.
+///
+///  * Default font-family for `Android`,`Fuchsia` and `Linux` is `Roboto`.
+///  * Default font-family for `iOS` is `.SF UI Display`/`.SF UI Text`.
+///  * Default font-family for `MacOS` is `.AppleSystemUIFont`.
+///  * Default font-family for `Windows` is `Segoe UI`.
```

Comment:
For optimal grammar correctness, `The default font-family for...`

Reply:
done.